cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_index_page.html

```diff
@@ -47,7 +47,7 @@
                 {%- endfor %}
             </div>
         </section>
-        <div class="block block__padded-top block__border-top u-mt0 u-mb30">
+        <div class="block block__padded-top u-mt0 u-mb30">
```

Comment:
While in here, you could update `u-mt0` to be `block__flush-top`, which would be the more "standard" way of removing the top margin.

Reply:
Done.